packages/components/src/components/toast/toast/toast-manager.ts

```diff
@@ -68,11 +71,11 @@ export class ToastManagerClass {
 
   public getToast(): ToastMessage {
     this.startTimeout();
-    return this.messages[0];
+    return this.message ? this.message : undefined;
```

Comment:
If it is undefined, this would return undefined directly.
```suggestion
    return this.message;
```